custom/icds_reports/tests/agg_tests/outputs/child_health_monthly_2017-04-01_sorted.csv

```diff
@@ -1203,7 +1203,7 @@ a7,4cb10cb5-6e3e-4076-a396-912c793e6c5a,2017-04-01,65,1,1,0,,,0,,1,23,0,0,0,0,0,
 a7,5147a42c-0d4d-4ceb-a051-183f53e896c7,2017-04-01,66,1,1,0,,,0,,1,18,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,,,,1,0,0,0,0,0,,,,,1,1,0,2011-10-16,M,72,sc,no,no,yes,,,,0,,0,,0,Name 1807,सीमा,,,,0,s7,st2,2016-12-22
 a7,518d3d0b-2fb2-4421-9446-1cdb1666524c,2017-04-01,52,1,1,1,normal,normal,1,,1,13,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,,13.5000000000000000,,0,0,0,0,0,0,unmeasured,unknown,unknown,unmeasured,1,1,0,2012-12-25,F,60,obc,no,no,yes,,,,0,,0,,0,Name 2267,Bindu,,,,0,s7,st2,2016-12-21
 a7,58181672-f483-49de-942f-7da022b5a3fc,2017-04-01,66,1,1,0,,,0,,1,19,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,,,,0,0,0,0,0,0,,,,,1,1,0,2011-10-16,M,72,obc,no,no,yes,,,,0,,0,,0,Name 2531,बब्बीता,,,,0,s7,st2,2016-12-27
-a7,625adb33-c67e-4151-93c7-64f28c988388,2017-04-01,46,1,1,0,,,0,,0,,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,,,0,0,0,0,0,0,,,,,0,0,0,2013-06-06,M,48,sc,no,no,yes,,,,0,,0,,0,Name 1783,रेरवा,,,,0,s7,st2,2016-12-22
+a7,625adb33-c67e-4151-93c7-64f28c988388,2017-04-01,46,1,1,1,moderately_underweight,moderately_underweight,1,,1,22,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,,11.8000000000000000,,0,0,0,0,0,0,unmeasured,unknown,unknown,unmeasured,1,1,0,2013-06-06,M,48,sc,no,no,yes,,,,0,,0,,0,Name 1783,रेरवा,,,,0,s7,st2,2016-12-22
```

Comment:
Why only this haas changed? It looks this was marked as migration previously and not with this new changed is not_migrated now. 
Can you verify two things?
1. how many child cases are migrated in local test data right now.
2. How many of them change their migration status because of this change?

@ShubhamBansal1997